app/src/main/java/com/gholem/moneylab/features/chart/adapter/item/ChartItem.kt

```diff
@@ -1,15 +1,21 @@
 package com.gholem.moneylab.features.chart.adapter.item
 
+import com.gholem.moneylab.domain.model.TransactionModel
+
 sealed class ChartItem {
     data class Category(
-        val categpry: ChartCategoryModel
+        val categpry: TransactionModel
```

Comment:
If `category` data is enough then change the object type to category instead of whole `TransactionModel`, otherwise change the name to `transactionModel`